### DISCUSSION TITLE
Stellarium licencing

### DISCUSSION THREAD

**Discussion by SanjeevMansu** [Stellarium/stellarium#2499] [Q&A] [answered]
Hi, I am Sanjeev From India. I am doing research in constellation. May I take help of Stellarium in my research work (Ph.D). Generally, I just want to take the Screen Shots of constellations with lines and drawings for my PhD thesis.

**Comment by gzotti** [comment, marked as answer]
In general screenshots from Stellarium are free to use. However, license conditions for the Sky Cultures are in the hands of the respective contributors. First, please update to the latest version. When you select a skyculture, the bottom of the description text show its license, like CC-BY-SA or CC-BY-NC-ND. You must follow these licenses and give credit to the skyculture authors where applicable. For academic work this may be unproblematic. If you later sell your PhD as book, this may exclude the use of images from skycultures published under a NC license.
